<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="page_view">
      <widget class="QWidget" name="page">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>391</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>Test area</string>
        </property>
        <widget class="QPushButton" name="sendBtn">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>send</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="sending_message">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>104</width>
           <height>70</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>54</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>request</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="recived_message">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>171</width>
           <height>251</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>50</y>
           <width>54</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>responce</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
